wca-regulations.md

```diff
@@ -87,7 +87,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - 2i1d) Food and drink.
     - 2i2) Competitors may use cameras at the solving station at the discretion of the WCA Delegate, but the following restrictions apply from the start of the attempt until the competitor stops the solve. Penalty for breaking a restriction: disqualification of the attempt (DNF).
         - 2i2a) Each camera monitor must be blank or out of sight of the competitor (see [Regulation A5b](regulations:regulation:A5b)).
-        - 2i2b) The competitor must not use (e.g. operate, wear) any active camera. Exception: the competitor may wear a camera mounted on their head, as long as it is out of their sight and it is clear that they are not interacting with it (apart from wearing it).
+        - 2i2b) The competitor must not use (e.g. operate) any active camera. Exception: the competitor may hold or wear a camera anywhere, as long as the camera's monitor is out of their sight and it is clear that they are not interacting with it (apart from wearing or holding it).
```

Comment:
```suggestion
        - 2i2b) The competitor must not use (e.g. operate) any active camera. Exception: the competitor may hold or wear a camera anywhere, as long as the camera's monitor is out of their sight or blank and it is clear that they are not interacting with it (apart from wearing or holding it).
```

Reply:
'anywhere' reads somewhat awkwardly and only makes sense in the context of updating the previous regulation rather than having a regulation that stands on its own feet. If this context is considered necessary we could either flag this word for future removal or remove it now and add a 'previously...' guideline.